chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.spec.ts

```diff
@@ -81,9 +87,31 @@ describe("activityMiddleware test", () => {
             }
         };
 
-        const results = createActivityMiddleware()()(next)(args);
+        const results = createActivityMiddleware(renderMarkdown)()(next)(args);
         expect(TelemetryHelper.logActionEvent).toHaveBeenCalledTimes(0);
-        expect(results().props?.dangerouslySetInnerHTML?.__html).toEqual("&lt;img src=&#39;&#39; onerror=&quot;alert(&#39;XSS attack&#39;)&quot;&#x2F;&gt;");
```

Comment:
please put this test back, it was part of an security sev2 incidence

Reply:
We have removed escapeHtml() because of which html element string were displayed as plain text in chat window
I referred https://github.com/preactjs/preact-render-to-string/issues/176 https://stackoverflow.com/questions/71159839/decode-string-in-react
Because of which output is not coming as encoded string so we have removed that step and checked if onerror=\"alert('XSS attack') is not present in output string .

Reply:
this one make me nervous... since we are not longer escaping, then when markdown is off, is there a risk of allowing any bad script to be injected as is?,

Reply:
No it wont be injected, I have checked in both cases when markeddown is on/off we are making sure that bad script (onerror=\"alert('XSS attack')) is not present in html string.

<img width="606" alt="image" src="https://github.com/microsoft/omnichannel-chat-widget/assets/128818969/00ec2c9e-6975-4574-86aa-7ff469e92cc8">